app/(afterLogin)/(employeeInformation)/employees/manage-employees/[id]/_components/job/addEmployeeJobInfrmation/index.tsx

```
@@ -6,47 +6,96 @@ import WorkScheduleForm from '../../../../_components/allFormData/workScheduleFo
+  const setModalWidth = isNavBarModal
+    ? setNavBarJobInfoModalWidth
+    : setEmployeeJobInfoModalWidth;
+
   const handleClose = () => {
-    setIsAddEmployeeJobInfoModalVisible(false);
-    setEmployeeJobInfoModalWidth(null);
+    setIsModalVisible(false);
+    setModalWidth(null);
   };
 
   const createTsks = (values: CreateEmployeeJobInformationInterface) => {
```

ESLint

[error] 'values' is defined but never used.

(@typescript-eslint/no-unused-vars)